### DISCUSSION TITLE
Start with Elastic Fleet now?

### DISCUSSION THREAD

**Discussion by adamphetamine** [Security-Onion-Solutions/securityonion#9938] [Unsupported Versions] [answered]
Wanting to get a head start on the move to Elastic Fleet, I note that it's possible to install on current version v2.3.220.

Not looking for an ironclad commitment, but is it likely to cause issues in the future?

ie. If you do this now you won't be able to upgrade to v2.4 ...

**Comment by InfosecGoon** [comment, marked as answer]
It's not something that we've tested in 2.3 -- I would recommend waiting for the 2.4 beta, or at the least confining your experimentation to a test environment.

**Comment by adamphetamine** [comment]
Cool, will do- thanks for the quick answer.
